snippets/main-pagination.liquid

```diff
@@ -1,7 +1,10 @@
-{%- if last > 1 -%}
+{% assign start = current | minus: 2 %}
+{% assign end = current | plus: 2 %}
+
+{% if last > 1 %}
   <div class='yc-pagination-wrapper'>
     <a
-      class='paginate-btn-prev {% if current <= 1 %} disabled {%- endif %}'
+      class='paginate-btn-prev {% if current <= 1 %} disabled {% endif %}'
```

Comment:
the disabled style is not included in the design you should double check it

Reply:
okay i'll double check with the product if we keep it or not